sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -67,6 +68,7 @@ If you havent created any consumer groups explicitly, then use the name of the d
 | In v2                                          | Equivalent in v5                                                 | Sample |
 |------------------------------------------------|------------------------------------------------------------------|--------|
 | `EventHubClient.receive()` and `EventHubClient.receiveBatch()`                       | `EventHubConsumerClient.subscribe()`                               | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/receiveEvents.ts) |
+| Consumer group was optional | Consumer group is mandatory | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/receiveEvents.ts) |
```

Comment:
This is already covered in the Client constructor section

https://github.com/Azure/azure-sdk-for-js/blob/081f98e03f8de6ef44f311d4ec60516b51aec059/sdk/eventhub/event-hubs/migrationguide.md#client-constructors

Reply:
Wires crossed, removed.